docs/config/node.rst

```diff
@@ -205,6 +208,93 @@ Ports
   range is used. If this is set to an integer value it is considered as an
   explicit single port.
 
+Advanced TCP settings
+---------------------
+
+Any interface that uses TCP (PG/HTTP/transport) shares the following settings:
```

Comment:
minor: Personally, I would expand to `Postgres wire, HTTP & Transport protocols`

Reply:
fine with that wording too - pushed a fixup